src/prompts/boilerplate.js

```diff
@@ -0,0 +1,82 @@
+const inquirer = require("inquirer");
+const questions = require("./questions");
+
+class Boilerplate {
+	constructor() {
+		this.prompt = inquirer.createPromptModule();
+		this.answers = {};
+		this.questions = questions;
+	}
+
+	async init() {
+		await this.getName();
+		await this.getPath();
+		await this.getDetails();
+		await this.getConfirmation();
+
+		return this.answers;
+	}
+
+	async getName () {
+		const response = await this.prompt([this.questions.name]);
+		let sanitized = response.name
+						.replace(/[A-Z]/g, letter => `-${letter.toLowerCase()}`) // lowers capitalized letters, adds '-' before each (newComponent => new-component)
```

Comment:
If the first letter of `response.name` is a capital, it would get a `-` added before it.
`Oa.replace(/[A-Z]/g, letter => `-${letter.toLowerCase()}`) // returns "-oa"`
Perhaps we should use lodash's kebab function instead of making our own?